index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+    <title>
+    Ariel's Portfolio
+    </title>
+    <section id="one">
+        <img class= "logo alignment" src="avatar.jpg">
+        <nav class="width">
+            <ul>
+                <li><a href="#one">Home</a></li>
+                <li><a href="#two">About Me</a></li>
+                <li><a href="#three">Portfolio</a></li>
+                <li><a href="#four">Contact</a></li>
+            </ul>
+            <div class="hamburger"></div>
+            <div class="hamburger"></div>
+            <div class="hamburger"></div>
+        </nav>
+    </section>
+
+    <section id="two" class="width alignment">
+        <div class="container">
+            <h1>About Me</h1>
+            <div class="about-me-text">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+                sed do eiusmod tempor incididunt ut labore et dolore magna
+                aliqua. Nulla aliquet enim tortor at auctor urna nunc id.
+                At augue eget arcu dictum varius duis at. Amet risus nullam
+                eget felis eget. Eget egestas purus viverra accumsan in nisl
+                nisi. Elit scelerisque mauris pellentesque pulvinar
+                pellentesque habitant morbi tristique. Dapibus ultrices in
+                iaculis nunc sed augue lacus viverra vitae. Eget velit aliquet
+                sagittis id. Eu nisl nunc mi ipsum. Eu lobortis elementum
+                nibh tellus molestie nunc non blandit massa. Feugiat in ante
+                metus dictum at tempor commodo. Magna fringilla urna porttitor
+            </div>
+            <img class="photo" src="avatar.jpg">
+        </div>
+    </section>
+
+<section id="three" class="width alignment">
```

Comment:
indentation needs to be in line with other sections (also id + class names need work)